abb_hardware_interface/src/abb_hardware_interface.cpp

```diff
@@ -216,10 +216,9 @@ std::vector<hardware_interface::StateInterface> ABBSystemHardware::export_state_
     {
       for (auto& joint : unit.joints)
       {
-        // TODO(seng): Consider changing joint names in robot description to match what comes
-        // from the ABB robot description to avoid needing to strip the prefix here
-        const auto pos = joint.name.find("joint");
-        const auto joint_name = joint.name.substr(pos);
+        const auto joint_name = joint.name;
+        RCLCPP_INFO_STREAM(LOGGER, "Unstripped name: " << joint.name);
```

Comment:
Are you sure you want this as loginfo? If yes, at least make it more descriptive.

Reply:
Oh thanks. This stayed from debugging ...